app/components/sdg/goals/targets_component.html.erb

```diff
@@ -0,0 +1,23 @@
+<section class="targets">
+  <ul class="tabs" id="target_tabs" data-deep-link="true" data-tabs>
+    <li class="tabs-title is-active">
+      <%= link_to SDG::Target.model_name.human(count: :other).upcase_first, "#tab_global_targets" %>
+    </li>
+    <li class="tabs-title">
+      <%= link_to SDG::LocalTarget.model_name.human(count: :other).upcase_first, "#tab_local_targets" %>
+    </li>
+  </ul>
+
+  <div class="tabs-content" data-tabs-content="target_tabs">
+    <% [global_targets, local_targets].each do |targets| %>
+      <div class="tabs-panel <%= active(targets) %>" id="tab_<%= type(targets) %>_targets">
+        <dl>
```

Comment:
Right now, when there are no local targets, we just get blank content when clicking on the "local targets" link. How about displaying a message in this case, like we do in other sections (milestones and proposal notifications in a proposal's view come to mind).